creme/creme_core/templates/creme_core/forms/widgets/date-range-select.html

```diff
@@ -1,4 +1,4 @@
-{% load i18n %}<span class="{{widget.class}}" widget="{{widget.widget_type}}" date_format="{{widget.date_format}}">
+{% load i18n %}<span class="{{widget.class}}" widget="{{widget.widget_type}}" format="{{widget.date_format}}">
```

Comment:
Why not "data-format" ?

Reply:
I would like but the current widgets does not work that way "out of the box". If a little hack is not an issue I do prefer the data-format option